### PR TITLE
Fix SCTP stream option bugs in sctp_test

### PR DESCRIPTION
**Pull request by jiachenjun**
current sctp_test can send packets with multiple stream,
But we found bugs in sctp_test stream option.
if we configure multiple stream number > 10.
the sctp_test will failed when send packets.

### CODE REVIEW

File: src/apps/sctp_test.c [hunk 1]
```diff
@@ -612,7 +612,17 @@ int socket_r(void)
 		fprintf(stderr, "SCTP_EVENTS: error: %d\n", error);
 		exit(1);
 	}
-
+        if (max_stream > 0) {
+        	struct sctp_initmsg initmsg;
+        	memset(&initmsg, 0, sizeof(struct sctp_initmsg));
+        	initmsg.sinit_num_ostreams = max_stream;
+                int i
```
Comment (marceloleitner) [inline src/apps/sctp_test.c]: Please remove this line, this variable is not used.

File: src/apps/sctp_test.c [hunk 2]
```diff
@@ -612,7 +612,17 @@ int socket_r(void)
 		fprintf(stderr, "SCTP_EVENTS: error: %d\n", error);
 		exit(1);
 	}
-
+        if (max_stream > 0) {
+        	struct sctp_initmsg initmsg;
+        	memset(&initmsg, 0, sizeof(struct sctp_initmsg));
+        	initmsg.sinit_num_ostreams = max_stream;
+                int i 
+                error = setsockopt(sk, IPPROTO_SCTP, SCTP_INITMSG, &initmsg, sizeof(struct sctp_initmsg));
```
Comment (marceloleitner) [inline src/apps/sctp_test.c]: And adjust the indentation here. There is an extra whitespace and the line is too long.

File: src/apps/sctp_test.c (hunk 1 above)
Comment (jiachenjun) [inline src/apps/sctp_test.c]: OK, done

File: src/apps/sctp_test.c (hunk 2 above)
Comment (jiachenjun) [inline src/apps/sctp_test.c]: Done.

**Comment by jiachenjun** [comment]
comments fixed, please review again. thanks.

**Comment by marceloleitner** [comment]
Squashed and merged. Next time just please be more careful with whitespaces for indentation. Thanks.

**Pull request merged**